File: src/main/java/roto/win/gamjaserver/listeners/ChatListener.java
package roto.win.gamjaserver.listeners;

import io.papermc.paper.event.player.AsyncChatEvent;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import roto.win.gamjaserver.CustomChatRenderer;

public class ChatListener implements Listener {

    @EventHandler
    public void onChat(AsyncChatEvent e) {
        e.renderer(new CustomChatRenderer());
    }
}

File: src/main/kotlin/win/roto/gamjaserver/listeners/ChatListener.kt
package win.roto.gamjaserver.listeners

import io.papermc.paper.event.player.AsyncChatEvent
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import win.roto.gamjaserver.CustomChatRenderer

class ChatListener : Listener {
    @EventHandler
    fun onChat(event: AsyncChatEvent) {
        event.renderer(CustomChatRenderer())
    }
}